### ISSUE TITLE
Hack the HACKnight

### ISSUE THREAD

**Issue by loleg** [loleg/dinacon-hacknight#1]
This is a meta issue to kick off some public discussions as we prepare to run the third [DINAcon Hacknight](https://dinacon.ch/en/hacknight) this Friday. We know: it's a short event, starting at 18:00, competing with the apéro and everyone's desire to go home after the big conference. There's a small budget to cover drinks and snacks and any other participant needs, but as nobody really takes much time to plan and organize, it kind of 'just happens'. The Hacknight, however, is *not* an afterthought. **There is a core of practitioners in the open source community that prioritizes hacking over slacking. As in, getting our hands dirty, fixing issues, contributing knowledge, helping others. It's a long tradition, foundational to the open source world as we know it.** Drinking fizzy beverages with code buddies and looking through stack traces may not be everyone's idea of a good time, so this Issue is here for any of you who have bright ideas or how else we could spend the time / resources / good will associated with the DINAcon Hacknight and events like it.

TL;DR

- let's make the Hacknights great again!
- connect to other similar events in the vicinity
- document and fix issues in our platforms (Dribdat, Mattermost, Git, ...)

For more background see my [2017 blog](https://blog.datalets.ch/034/) and [2018 posts](https://forum.opendata.ch/t/19-10-dinacon-hacknight-2018/438/1), as well as a [summer hack](https://github.com/loleg/earnopendata) on the related topic of making open projects sustainable.

_This issue is part of [DINAcon HACKnight 2019](https://hacknight.dinacon.ch/project/25)_
